src/tools/context/getInitialContextTool.ts

```diff
@@ -18,10 +17,9 @@ export function hasInitialContext(): boolean {
 }
 
 async function tool(_params: Params) {
-  const [config, datasets, schema, embeddings, releases] = await Promise.all([
```

Comment:
clients seem to always call schema tool anyway, so avoiding filling up the context window